megamek/src/megamek/common/LandAirMech.java

```diff
@@ -127,6 +127,13 @@ public static int getAeroLocation(int loc) {
     private int straightMoves = 0;
     private int altLoss = 0;
     private int altLossThisRound = 0;
+    
+    //Autoejection
+    private boolean autoEject = true;
+    private boolean condEjectAmmo = true;
```

Comment:
Mech already defines these fields with public accessors. There is no reason to give LandAirMech separate copies.

Reply:
Fixed